azure.dockerfile

```diff
@@ -0,0 +1,31 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
```

Comment:
Let's modified it when we need it.
> I wonder if it would make the dockerfile more flexible if the base image name (or mariner version) is passed as an ARG. Especially when we will have mariner version 3.0 available.
I could not find a doc to tell me whether there will be Azure Linux 2 or not.
Currently, Azure Linux is Mariner 2. I don't know if there will be Azure Linux 2.